db/errors/error-handling.js

```diff
@@ -1,3 +1,20 @@
+exports.customErrorHandler = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+exports.postgresErrorHandler = (err, req, res, next) => {
+  switch (err.code) {
+    case "22P02":
+      res.status(400).send({ msg: "Bad request" });
+      break;
+    default:
+      next(err);
+      break;
```

Comment:
A matter of opinion, but do be consistent in your conditional logic. If/else and then switch case doesn't show a consistent style.